compiler-rt/test/pfp/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(PFP_TESTSUITES)
+set(PFP_TEST_DEPS
+  ${SANITIZER_COMMON_LIT_TEST_DEPS})
+
+set(PFP_TEST_ARCH ${X86_64} ${ARM64})
+
+configure_lit_site_cfg(
+  ${CMAKE_CURRENT_SOURCE_DIR}/lit.site.cfg.py.in
+  ${CMAKE_CURRENT_BINARY_DIR}/${CONFIG_NAME}/lit.site.cfg.py)
+
+list(APPEND PFP_TESTSUITES ${CMAKE_CURRENT_BINARY_DIR}/${CONFIG_NAME})
+
+add_lit_testsuite(check-pfp "Running the PointerFieldProtection tests"
+  ${PFP_TESTSUITES}
```

Comment:
This should be depending on lld conditionally to handle the case where lld is in LLVM_ENABLE_PROJECTS, look at what other tests are doing

https://github.com/llvm/llvm-project/blob/ad1ca5f4a2bc09f99fd82e5444f5da37c2985e97/compiler-rt/test/msan/CMakeLists.txt#L24

Reply:
Done. Thanks for the link.